defaults/main.yml

```diff
@@ -81,10 +81,19 @@ datadog_rpm_remove_keys: [4172A230]
 
 # Default yum repo and keys
 
+# By default, the role configures a yum repository for installing the Datadog agent package
+# Use the datadog_yum_repo_setup_enabled toggle to disable this configuration
```

Comment:
```suggestion
# By default, the role configures a yum repository for installing the Datadog agent package
# Use the datadog_yum_repo_setup_enabled toggle to disable this configuration
# WARNING: if you use this option, you will have to manually maintain the list of GPG signing keys used to sign the packages and update it as necessary by yourself
```
As your option setting wraps up the GPG key settings, we need an explicit warning that this feature is thus deactivated